CHANGES.rst

```diff
@@ -293,7 +294,7 @@ Internal Changes
 - Add ``<ind-menu>`` custom element for managing drop-down menus (:issue:`5896`, :pr:`5897`,
   thanks :user:`foxbunny`)
 - Allow plugins to add extra fields to the room booking form (:pr:`6126`, thanks :user:`VojtechPetru`)
-
+- Add ``<ind-date-picker>`` custom element (:pr:`6371`, thanks :user:`foxbunny`)
```

Comment:
```suggestion
- Add ``<ind-date-picker>`` custom element (:pr:`6371`, thanks :user:`foxbunny`)

```

otherwise i'll forget fixing this inconsistency before merging the PR ;)